src/pages/community/patterns/create-flows/index.mdx

```diff
@@ -3,367 +3,440 @@ title: Create flows
 description:
   To create is to generate a new resource. Creating is distinct from adding, but
   the two are often confused. To add a resource is to include a resource that
-  already exists.
+  already exists. This pattern focuses on how to place the user in a flow when
+  there's a need for creating a new resource.
 ---
 
 #### Maintainers:
 
-[Vikki Paterson](https://github.com/vikkipaterson),
-[Chloe Poulter](https://github.com/chloepoulterdesign)
+[Andrew Smith](https://w3.ibm.com/bluepages/profile.html?uid=4G8681897), 
+[Andrea Garcia](https://w3.ibm.com/bluepages/profile.html?uid=4G0746897)
 
-<PageDescription>
+## Overview
```

Comment:
I think it would be good to have in overview a `h3` "Creation flow types" sections that gives a quick high level summary of each format. Similar to what we do in the [dialog pattern](https://www.carbondesignsystem.com/patterns/dialog-pattern#dialog-types).

Reply:
I've added the table in, hopefully @andrea-gm can add the text for each row.